cmake/Modules/FindTPLCUDA.cmake

```diff
@@ -33,8 +40,8 @@ else()
   endif()
 
   include(FindPackageHandleStandardArgs)
-  find_package_handle_standard_args(TPLCUDA ${DEFAULT_MSG} FOUND_CUDART FOUND_CUDA_DRIVER)
+  find_package_handle_standard_args(TPLCUDA ${DEFAULT_MSG} FOUND_CUDA_DRIVER FOUND_CUDART)
```

Comment:
is there an actual reason for this or just a "I first deleted CUDART here and then readded it" situation?

Reply:
Nope you have it exactly right! I had intermediate changes and tried to put back the original state